### PR TITLE
Add-README-files-3/8

### PR DESCRIPTION
**Pull request by Leoriem-code**
### Describe your change:

The 3rd of the serie

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: arithmetic_analysis/in_static_equilibrium.py [hunk 1]
```diff
@@ -19,8 +19,6 @@ def polar_force(
     True
     >>> math.isclose(force[1], 7.0710678118654755)
     True
-    >>> polar_force(10, 3.14, radian_mode=True)
-    [-9.999987317275396, 0.01592652916486828]
```
Comment (CaedenPH) [inline arithmetic_analysis/in_static_equilibrium.py:23]: Can this change be reverted? Not sure why it was made

File: data_structures/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Data Structures
+
+There exist [many](https://en.wikipedia.org/wiki/List_of_data_structures) types of data structures, but some structures are more useful to keep some type of data over another type.
```
Comment (CaedenPH) [inline data_structures/README.md:3]:
```suggestion
[Many](https://en.wikipedia.org/wiki/List_of_data_structures) types of data structures exist, but some structures are more useful than others when storing certain types of data.
```

File: data_structures/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+The types used everywhere when programming is the boolean, the integer, the character and the floating point number. Those can lead to more advanced type, such as list, array, set, tuple, stack etc.
+
+(If this subject has your interest, you should do your own research because it is a vast subject. As often, the starting point should be the Wikipedia page.)
+
+* <https://en.wikipedia.org/wiki/Data_structure>
+* <https://www.tommyds.it/doc/benchmark.html>
```
Comment (CaedenPH) [inline data_structures/README.md:9]: Doesn't seem relevant

File: digital_image_processing/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Digital Image Processing
+
+Digital Image Processing (DIP) is when you are processing and applying changes to an image with a computer.
```
Comment (CaedenPH) [inline digital_image_processing/README.md:3]:
```suggestion
Digital Image Processing (DIP) is the process of applying changes to an image digitally
```

File: data_structures/README.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Data Structures
+
+There exist [many](https://en.wikipedia.org/wiki/List_of_data_structures) types of data structures, but some structures are more useful to keep some type of data over another type.
+The types used everywhere when programming is the boolean, the integer, the character and the floating point number. Those can lead to more advanced type, such as list, array, set, tuple, stack etc.
```
Comment (CaedenPH) [inline data_structures/README.md:4]: This line is wordy and doesn't make sense

**Review by CaedenPH — CHANGES_REQUESTED** [review]
I like the idea of adding README's to each directory, however the content here is insufficient an phrased incorrectly

@tianyizheng02 What is your opinion?

**Comment by tianyizheng02** [comment]
> I like the idea of adding README's to each directory, however the content here is insufficient an phrased incorrectly
> 
> @tianyizheng02 What is your opinion?

@CaedenPH I agree, the content needs to be significantly expanded upon and the grammar/wording is a bit questionable. Having directory READMEs is definitely good documentation so we'll just have to wait until someone else opens another PR contributing one.

**Pull request closed without merging**